jsuite/jrecipes/schema/mutations.py

```diff
@@ -1,9 +1,293 @@
 import graphene
 
-from jrecipes.models import Recipe
-from jrecipes.schema.types import RecipeInput, RecipeType
+from jrecipes.models import Category, Allergen, InstructionStep, InstructionSubStep, Ingredient, IngredientListItem, Recipe
+from jrecipes.schema.types import CategoryInput, CategoryType, AllergenInput, AllergenType, InstructionStepInput, InstructionStepType, InstructionSubStepInput, InstructionSubStepType, IngredientInput, IngredientType, IngredientListItemInput, IngredientListItemType, RecipeInput, RecipeType
 
+#################################################
+#               Category
```

Comment:
This is okay but didn't we discuss about a certain folder structure that makes this kind of workaround unnecessary? (this comment also goes for the models, types, queries I think)

Reply:
See issue #53